tsconfig.json

```
@@ -1,15 +1,11 @@
 {
+  "extends": "@vue/tsconfig/tsconfig",
```

Fully resolve the import as in the docs
```suggestion
  "extends": "@vue/tsconfig/tsconfig.json",
```